pkgs/gui-scale-applet/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchFromGitHub,
+  libcosmicAppHook,
+  rustPlatform,
+  just,
+  stdenv,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "gui-scale-applet";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "cosmic-utils";
+    repo = "gui-scale-applet";
+    rev = "9251253d6332820d267bf9d825e6af522ae63eb7";
+    hash = "sha256-1zCANfgWgDkSTvpvgxzve/ErGel2WF1RxIhv/EdIxxo=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-2yfeZKsakAeAtNcK8v7hqwMBm7o7HhiNU5mgPevhNvo=";
```

Forgot to include the updated cargo hash in the previous suggestion

```suggestion
  cargoHash = "sha256-m8Ap3aXnjwQ14boE+e3zrYSOimLK+NXjA+KO7uwXktQ=";
```